src/Tickets/Commerce/Ticket.php

```diff
@@ -181,6 +190,18 @@ class Ticket {
 	 */
 	public static $sale_price_end_date_key = '_sale_price_end_date';
 
+	/**
+	 * Sets the pending attendee count for a ticket id.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $ticket_id      The ticket ID.
+	 * @param int $attendee_count The number of attendees.
+	 */
+	public static function set_attendees_by_ticket( $ticket_id, $attendee_count ) {
+		static::$pending_attendees_by_ticket[ $ticket_id ] = $attendee_count;
+	}
```

Comment:
We need to improve the name of these variables and methods, as of right now this will cause confusion.

We need to allow attendees by ticket to be stored for each one of the possible statuses, which this will not allow.

So potentially store attendees by ticket and each status as a new index, so we can get all the counts per ticket ID.